plugin.video.jupiterbroadcasting/resources/language/Bulgarian/strings.po

```diff
@@ -0,0 +1,282 @@
+# XBMC Media Center language file
```

Comment:
As part of the language stuff adaptions for Krypton, please change the folder layout for language files. It should be:

`resources/language/resource.language.bg_bg/strings.po`


More info can be checked in the following links:

https://kodi.wiki/view/Language_support
https://github.com/xbmc/xbmc/tree/master/addons/skin.estuary/language/